chsdi/views/catalog.py

```diff
@@ -1,14 +1,87 @@
 # -*- coding: utf-8 -*-
 
 from pyramid.view import view_config
-from pyramid.httpexceptions import HTTPNotFound
+import networkx as nx
+from itertools import chain
+from pyramid.httpexceptions import HTTPInternalServerError, HTTPNotFound
 
 from chsdi.models.bod import Catalog
 from chsdi.lib.validation import MapNameValidation
 from chsdi.lib.sqlalchemy_customs import remove_accents
 from chsdi.lib.filters import filter_by_geodata_staging
 
 
+# Cutsom networkx like function
```

Comment:
Typo. Please adapte and merge when done. Thanks alot for this, @loicgasser and @mariavkourouni .